Tutorials/Members-Registration-And-Logins/index.md

```diff
@@ -0,0 +1,245 @@
+
+Now, we have our signup and login functionality - though there is one partial view we have not used yet, the Login Status.
+We created it as a Partial View, not as a Macro, for a singular reason - we are going to include it directly in a template, not in a property editor in the Content section.
+This partial will show the current status - whether the person is logged in or not.
+A good place to put it would be either the Master page, or a navigation component.
+
+![Content Template Name Field](images/v8-10-Login-Partial-In-Nav.png)
+
+With this partial in place, provided we put it in navigation or a layout page, we will be able to see the login status no matter on which page we currently are. It also allows the member to log out.
+Do keep in mind that you might need to style the partial a bit to make it fit with the rest of the site.
+
+![New Content Template](images/v8-11-Login-Partial-Frontend.png)
+
+## Member-only pages/Restricted access
+
+Now that we have the options to:
+-register a member
+-log in a member
+-check current login status
+-log out a member
+
+We can also go a bit further and specify which parts of our website should be accessible to logged in members.
+To do this, head on over to the Member section in the Backoffice, and create a new Member Group.
+
+![Edit Content Template](images/v8-12-New-Member-Group.png)
+
+Give the group a name, and save it. Next, move over to the Members folder, click on any of the Members you  have, and assign the newly created Member Group to them.
+
+![Create From Template](images/v8-13-Member-Group-Assign.png)
+
+Almost there. Save the member, move to the Content section and find the page you would like to restrict. Right-click on that content node, and find the option "Restrict Public Access".
+
+![Select Template](images/v8-14-Restrict-Public-Access.png)
+
+You will be able to restrict access to a specific member, or a specific group. Choose the latter option.
+On this menu, you will be able to select the group that will have access to the page, the page with the login form, and the page that would be displayed if selected content would be inaccessible to the chosen group.
+
+![Select Template](images/v8-15-Restrict-Public-Access-Details.png)
+
+Congratulations! With all of that set up, the page you chose will redirect the user to the Login page if they are not logged in, and if they are logged in - they will be able to see the page's contents.
+
+:::note
+The above approach relies on two Macro partial views and a non-Macro partial view. It is also possible to achieve the same result by working entirely with plain Partial Views, three Macros, and even plain HTML/Razor code copied from the Snippets into your Templates.
+:::
+
+However, with the above approach, members will not be assigned to any group automatically - for this to happen, we would need to write a bit of custom code.
+
+## Going further - creating a new SurfaceController
```

Comment:
How about using a heading here that's more related to the task that's being accomplished by following the steps?
Yes, you will be creating a new Surface Controller, but you will also be adding the functionality of automatically assigning new members to a specific member group.

Consider what would be most valuable for the reader to know!

```suggestion
## Going further - creating a new SurfaceController
```

Reply:
restructured headings

## Assigning new members to groups automatically

### Creating a new SurfaceController

### Adjusting the Registration partial to use the new controller